packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.service.js

```diff
@@ -0,0 +1,172 @@
+import find from 'lodash/find';
```

Comment:
Please use `https://github.com/ovh/manager/tree/develop/packages/manager/modules/product-offers` instead

Reply:
Thanks for letting me know about this component. I had a look at it and I see that the component is capable of displaying the various pricing for an addon. But our requirement is that we display different addons for the user to choose 1 from. Unfortunately, this component would not fit our requirement